list.go

```diff
@@ -271,6 +276,18 @@ NextFile:
 							p.IgnoredGoFiles = append(p.IgnoredGoFiles, fname)
 							continue NextFile
 						}
+						matchesPositive, err := regexp.MatchString("go\\d+\\.\\d+", b)
```

Comment:
compile this regex at the package level in a variable and re-use maybe? allows this to be written nicer/shorter at least.